generate-epub.sh

```diff
@@ -6,7 +6,7 @@ generate_from_stdin() {
 
   echo "Generating '$language' ..."
 
-  pandoc --metadata-file=epub-metadata.yaml --metadata=lang:$2 --from=markdown -o $1 <&0
+  pandoc --epub-metadata=epub-metadata.yaml --metadata=lang:$2 --from=markdown -o $1 <&0
```

Comment:
According to [Pandoc Docs](https://pandoc.org/MANUAL.html#option--epub-metadata), `--epub-metadata` expects a XML file containing the required metadata.

I tested your change locally and the resulting epub didn't contain any metadata, so this doesn't work as expected at least for me.

Reply:
Hi I am using pandoc 2.2.3.2

Reply:
It seems mine is older

Reply:
This is snapshot of my pandoc help : 
`
pandoc [OPTIONS] [FILES]
  -f FORMAT, -r FORMAT  --from=FORMAT, --read=FORMAT                    
  -t FORMAT, -w FORMAT  --to=FORMAT, --write=FORMAT                     
  -o FILE               --output=FILE                                   
                        --data-dir=DIRECTORY                            
                        --base-header-level=NUMBER                      
                        --strip-empty-paragraphs                        
                        --indented-code-classes=STRING                  
  -F PROGRAM            --filter=PROGRAM                                
                        --lua-filter=SCRIPTPATH                         
  -p                    --preserve-tabs                                 
                        --tab-stop=NUMBER                               
                        --track-changes=accept|reject|all               
                        --file-scope                                    
                        --extract-media=PATH                            
  -s                    --standalone                                    
                        --template=FILE                                 
  -M KEY[:VALUE]        --metadata=KEY[:VALUE]                          
  -V KEY[:VALUE]        --variable=KEY[:VALUE]                          
  -D FORMAT             --print-default-template=FORMAT                 
                        --print-default-data-file=FILE                  
                        --print-highlight-style=STYLE|FILE              
                        --dpi=NUMBER                                    
                        --eol=crlf|lf|native                            
                        --wrap=auto|none|preserve                       
                        --columns=NUMBER                                
                        --strip-comments                                
                        --toc, --table-of-contents                      
                        --toc-depth=NUMBER                              
                        --no-highlight                                  
                        --highlight-style=STYLE|FILE                    
                        --syntax-definition=FILE                        
  -H FILE               --include-in-header=FILE                        
  -B FILE               --include-before-body=FILE                      
  -A FILE               --include-after-body=FILE                       
                        --resource-path=SEARCHPATH                      
                        --request-header=NAME:VALUE                     
                        --self-contained                                
                        --html-q-tags                                   
                        --ascii                                         
                        --reference-links                               
                        --reference-location=block|section|document     
                        --atx-headers                                   
                        --top-level-division=section|chapter|part       
  -N                    --number-sections                               
                        --number-offset=NUMBERS                         
                        --listings                                      
  -i                    --incremental                                   
                        --slide-level=NUMBER                            
                        --section-divs                                  
                        --default-image-extension=extension             
                        --email-obfuscation=none|javascript|references  
                        --id-prefix=STRING                              
  -T STRING             --title-prefix=STRING                           
  -c URL                --css=URL                                       
                        --reference-doc=FILE                            
                        --epub-subdirectory=DIRNAME                     
                        --epub-cover-image=FILE                         
                        --epub-metadata=FILE                            
                        --epub-embed-font=FILE                          
                        --epub-chapter-level=NUMBER                     
                        --pdf-engine=PROGRAM                            
                        --pdf-engine-opt=STRING                         
                        --bibliography=FILE                             
                        --csl=FILE                                      
                        --citation-abbreviations=FILE                   
                        --natbib                                        
                        --biblatex                                      
                        --mathml                                        
                        --webtex[=URL]                                  
                        --mathjax[=URL]                                 
                        --katex[=URL]                                   
                        --gladtex                                       
                        --abbreviations=FILE                            
                        --trace                                         
                        --dump-args                                     
                        --ignore-args                                   
                        --verbose                                       
                        --quiet                                         
                        --fail-if-warnings                              
                        --log=FILE                                      
                        --bash-completion                               
                        --list-input-formats                            
                        --list-output-formats                           
                        --list-extensions[=FORMAT]                      
                        --list-highlight-languages                      
                        --list-highlight-styles                         
  -v                    --version                                       
  -h                    --help   
`

Reply:
There is no option as metadata-file .

Reply:
This is the snapshot of my pandoc help :
pandoc [OPTIONS] [FILES]
  -f FORMAT, -r FORMAT  --from=FORMAT, --read=FORMAT                    
  -t FORMAT, -w FORMAT  --to=FORMAT, --write=FORMAT                     
  -o FILE               --output=FILE                                   
                        --data-dir=DIRECTORY                            
                        --base-header-level=NUMBER                      
                        --strip-empty-paragraphs                        
                        --indented-code-classes=STRING                  
  -F PROGRAM            --filter=PROGRAM                                
                        --lua-filter=SCRIPTPATH                         
  -p                    --preserve-tabs                                 
                        --tab-stop=NUMBER                               
                        --track-changes=accept|reject|all               
                        --file-scope                                    
                        --extract-media=PATH                            
  -s                    --standalone                                    
                        --template=FILE                                 
  -M KEY[:VALUE]        --metadata=KEY[:VALUE]                          
  -V KEY[:VALUE]        --variable=KEY[:VALUE]                          
  -D FORMAT             --print-default-template=FORMAT                 
                        --print-default-data-file=FILE                  
                        --print-highlight-style=STYLE|FILE              
                        --dpi=NUMBER                                    
                        --eol=crlf|lf|native                            
                        --wrap=auto|none|preserve                       
                        --columns=NUMBER                                
                        --strip-comments                                
                        --toc, --table-of-contents                      
                        --toc-depth=NUMBER                              
                        --no-highlight                                  
                        --highlight-style=STYLE|FILE                    
                        --syntax-definition=FILE                        
  -H FILE               --include-in-header=FILE                        
  -B FILE               --include-before-body=FILE                      
  -A FILE               --include-after-body=FILE                       
                        --resource-path=SEARCHPATH                      
                        --request-header=NAME:VALUE                     
                        --self-contained                                
                        --html-q-tags                                   
                        --ascii                                         
                        --reference-links                               
                        --reference-location=block|section|document     
                        --atx-headers                                   
                        --top-level-division=section|chapter|part       
  -N                    --number-sections                               
                        --number-offset=NUMBERS                         
                        --listings                                      
  -i                    --incremental                                   
                        --slide-level=NUMBER                            
                        --section-divs                                  
                        --default-image-extension=extension             
                        --email-obfuscation=none|javascript|references  
                        --id-prefix=STRING                              
  -T STRING             --title-prefix=STRING                           
  -c URL                --css=URL                                       
                        --reference-doc=FILE                            
                        --epub-subdirectory=DIRNAME                     
                        --epub-cover-image=FILE                         
                        --epub-metadata=FILE                            
                        --epub-embed-font=FILE                          
                        --epub-chapter-level=NUMBER                     
                        --pdf-engine=PROGRAM                            
                        --pdf-engine-opt=STRING                         
                        --bibliography=FILE                             
                        --csl=FILE                                      
                        --citation-abbreviations=FILE                   
                        --natbib                                        
                        --biblatex                                      
                        --mathml                                        
                        --webtex[=URL]                                  
                        --mathjax[=URL]                                 
                        --katex[=URL]                                   
                        --gladtex                                       
                        --abbreviations=FILE                            
                        --trace                                         
                        --dump-args                                     
                        --ignore-args                                   
                        --verbose                                       
                        --quiet                                         
                        --fail-if-warnings                              
                        --log=FILE                                      
                        --bash-completion                               
                        --list-input-formats                            
                        --list-output-formats                           
                        --list-extensions[=FORMAT]                      
                        --list-highlight-languages                      
                        --list-highlight-styles                         
  -v                    --version                                       
  -h                    --help

Reply:
Maybe try to update your version.

Reply:
I have tried sudo apt-get update and sudo apt install pandoc but it is saying "pandoc is already the newest version (2.5-3build2)". Could you help me to know how to update package on ubuntu

Reply:
I understand problem is in my system . Thanks for clarification . How to close this pull request without merging ?

Reply:
There should be a Button at the bottom to close it.

Well if you just want the epub, I guess I could send it to you :D Drop me a message via Twitter or E-Mail, we'll figure it out.